### PR TITLE
Instructions for new daria instance setup, redux

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Somehow the doc in #1067 got eaten. This re-adds it and finishes it off. 

This pull request makes the following changes:
* re-adds instance deployment instructions

I'll keep this floating until we're all clear.

### CODE REVIEW

File: docs/SETTING_UP_A_NEW_INSTANCE.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- `mongodb` (database from MONGODB_URI)
+- `mongohost` (hostname from MONGODB_URI)
+- `mongopass` (password from MONGODB_URI)
+- `mongoport` (port from MONGODB_URI)
+- `mongouser` (user from MONGODB_URI)
+- `NEW_RELIC_LICENSE_KEY` (Get from New Relic APM)
+- `RAILS_LOG_TO_STDOUT` (set to `true`)
+- `RAILS_SERVE_STATIC_FILES` (set to `true`)
+- `SITE_URL` (The URL, without http, your CMs will go to. e.g. `app.myabortionfund.org`)
+- `SKYLIGHT_AUTH_TOKEN` (get it from Skylight)
```
Comment (lomky) [inline docs/SETTING_UP_A_NEW_INSTANCE.md]: Do we need the external pledge & insurances envs here?

Comment (colinxfleming) [inline docs/SETTING_UP_A_NEW_INSTANCE.md]: fantastic catch! I'll add a note to the optional section about that

**Comment by colinxfleming** [comment]
This is good to go from my end -- it's an accurate reflection of what I did. I'll set a reminder to bump this again before we declare FINAL VICTORY on the project, but it's an accurate state-of-the-app reflection right now. I think. (@lomky caught a pretty big oversight, thank you!) 

@lomky @DarthHater @cfinucane I'm tagging any and all of you in on this to make sure the directions make sense before I merge.

File: docs/SETTING_UP_A_NEW_INSTANCE.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+## By the way
+
+DCAF has a lot of the surrounding infrastructure set up in such a way that it's easy to add new instances to our set. So please reach out to us if you're interested in skipping a lot of the tedious technical setup here.
+
+## Main steps
+
+### Create a new heroku app
+
+* Create a new Heroku app (if in DCAF pipeline: named xxx-fundname)
```
Comment (lomky) [inline docs/SETTING_UP_A_NEW_INSTANCE.md]: What does "DCAF Pipeline" mean? Might be worth spelling that out in a footnote.

Comment (colinxfleming) [inline docs/SETTING_UP_A_NEW_INSTANCE.md]: good flag on this! Refers to the heroku pipeline we have set up, but I'll make a tweak to this to clarify for sure.

**Pull request merged**